packages/react/src/utils/isServer.ts

```diff
@@ -0,0 +1,3 @@
+export const isServer = () => {
+  return typeof window === 'undefined' && typeof global !== 'undefined'
```

Comment:
I don't know why we should check global. could you explain it please?

No global checking libraries references
- usehoos-ts: https://usehooks-ts.com/react-hook/use-isomorphic-layout-effect
- react-use: https://github.com/streamich/react-use/blob/master/src/useIsomorphicLayoutEffect.ts

Reply:
I think, if in node env or server env, `this` is global so this check is equal to `typeof window === 'undefined'`

Reply:
![Uploading image.png…]()

Reply:
I can accept semantical cause. but I think if we don't check global in this case, we can get better performance. because we are making most fundamental library so this isClient function will be recalled many time. so I don't want to give up our better performance if we can

I asked this to slash library too. https://github.com/toss/slash/issues/352

Every libraries I checked don't check global to know it is server or client. so I'm very curious your and slash's intention

Reply:
I agree. this function is called every time in bundler position. 
So this expression called inline, rather than export.

Reply:
[chore: delete isClient, isServer function](https://github.com/suspensive/react/pull/239/commits/6a2514d5411688131230be159edb11da6fd5ffc4)

I delete it !! many thanks 🚀

Reply:
Thanks for accepting my opinion, Let's improve this if we need to improve it after